python/py-gpsoauth/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-gpsoauth
 version             1.0.2
-revision            0
+revision            1
```

Comment:
why does this need a `revision` increase? It presumably did build before for the already listed Python versions, but even if  not you are only changing build dependencies here so normally there is no reason to increase the `revision` field.

Reply:
I have changed the build backed from `setuptools` to `poetry`, that makes different build dependnecy.

Perhabs it overenginereg and I'll decreased it back.

Reply:
Exactly, and you don't need a `revision` increase if you only change build dependencies.